Components/Avatar/avatar.css

```diff
@@ -0,0 +1,114 @@
+@import url("../../style/reset.css");
+@import url("../../style/color.css");
+@import url("../../style/common.css");
+
+.avatar{
+  border-radius: 50%;
+  width: 48px;
+  height: 48px;
+}
+
+.xs {
+  width: 24px;
+  height: 24px;
+  font-size: 7px;
+}
+
+.sm {
+  width: 32px;
+  height: 32px;
+  font-size: 1rem;
+}
+
+.md {
+  width: 48px;
+  height: 48px;
```

Comment:
px to rem

Reply:
It would be good if you change the px into rem for the responsiveness